TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/ReferencesEditor.js

```diff
@@ -26,10 +26,17 @@ define(
 			// Minimum amount of characters to trigger search
 			threshold: 2,
 
+			// The path to a node that defines the starting point for the reference search
+			startingPoint: null,
+
 			didInsertElement: function() {
-				var that = this;
+				var that = this,
+					currentQueryTimer = null;
+
+				if (this.get('startingPoint') === null) {
```

Comment:
maybe a less strict check in case it's an empty string? less error prone if people try to unset it